t/page/needed.rb

```diff
@@ -0,0 +1,39 @@
+require_relative '../../lib/page/needed'
+require 'minitest/autorun'
+require 'octokit'
+require 'pry'
+require 'vcr'
+
+describe 'Needed' do
+  VCR.configure do |config|
+    config.cassette_library_dir = 't/fixtures/vcr_cassettes'
+    config.hook_into :webmock
+    config.ignore_request do |req|
+      req[:uri].include? 'https://cdn.rawgit.com/everypolitician/everypolitician-data'
+    end
+  end
+
+  subject do
+    Page::Needed.new
+  end
+
+  it 'should return a list of countries labeled to_find' do
```

Comment:
This (both in name, and in what it's actually testing) is a little bit too coupled to the internal implementation of the method. What you should be testing is that we can get a list of the countries that we still don't even need a source for. That is currently achieved by querying labels, but that should be irrelevant to the test. If we entirely change how that works (e.g. by looking them up in a Google Spreadsheet) we should only need to change the code — not the tests.